ci/dox.sh

```diff
@@ -22,7 +22,10 @@ dox() {
   rm -rf target/doc/$arch
   mkdir target/doc/$arch
 
-  rustdoc --target $target -o target/doc/$arch src/lib.rs --crate-name stdsimd
+  cargo clean
+  cargo build --target $target
+
+  rustdoc --target $target -o target/doc/$arch src/lib.rs --crate-name stdsimd --library-path target/$target/debug/deps
```

Comment:
Now `rustdoc` needs a build of the `coresimd` to generate the docs for `stdsimd`. I haven't taken a detailed look at the docs, and haven't looked at the output of this at all. `cargo doc` seems to work just fine for `stdsimd`, wrapping the docs of both crates nicely, although some links on `coresimd` are still pointing to `stdsimd`.